src/main/java/org/opentripplanner/routing/algorithm/raptoradapter/transit/FlexAccessEgressAdapter.java

```diff
@@ -29,6 +29,13 @@ public int latestArrivalTime(int requestedArrivalTime) {
     return flexAccessEgress.latestArrivalTime(requestedArrivalTime);
   }
 
+  @Override
+  public boolean hasOpeningHours() {
+    // TODO OTP2: THIS SHOULD BE IMPLEMENTED SO WE CAN FILTER FLEX ACCESS AND EGRESS
+    //            IN ROUTING, IT IS SET TO TRUE NOW TO ASSUME ALL FLEX HAS OPENING HOURS
```

Comment:
Per now, this is only used to filter access/egress paths before running the raptor search. The filtering happens in a ParetoSet in AccessEgressFunctions. All access/egress with opening hours are kept, only ones with no restrictions are allowed to dominate another one. Hence; setting this to true is not optimal, but do not cause any errors or lost solutions.